tests/cross_functional/flowtest/pvc_snapshot_and_clone/test_pgsql_pvc_snapshot_and_clone.py

```
@@ -79,26 +83,43 @@ def test_encrypted_pvc_snapshot_and_clone(
+        logger.test_step("Create encrypted storage class for RBD")
         self.sc_obj = storageclass_factory(
             interface=CEPHBLOCKPOOL,
             encrypted=True,
             encryption_kms_id=self.vault.kmsid,
         )
+        logger.info(f"Created encrypted storage class: {self.sc_obj.name}")
 
-        # Create ceph-csi-kms-token in the tenant namespace
+        logger.test_step(f"Create Vault CSI KMS token in namespace: {BMO_NAME}")
         self.vault.vault_path_token = self.vault.generate_vault_token()
+        logger.info(f"Generated Vault token: {self.vault.vault_path_token[:20]}...")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)